openapi/kas-fleet-manager-private-admin.yaml

```diff
@@ -312,10 +308,6 @@ components:
           type: string
         kafka_ibp_version:
           type: string
-        kafka_storage_size:
```

Comment:
We could consider bumping the API version https://github.com/valeriiashapoval/kas-fleet-manager/blob/2b86e31a2fcea4b28dc7f0bd4e3708f38d3d558b/openapi/kas-fleet-manager-private-admin.yaml#L5